src/main/resources/config/config.yaml

```diff
@@ -13,7 +13,7 @@ logging:
         threshold: ALL
         timeZone: UTC
         target: stdout
-        logFormat: "[%d{yyyy-MM-dd HH:mm:ss.SSS}] [%thread] %level %logger{15} [requestID=%X{X-Request-Id}] - %msg %n"
+        logFormat: "[%d{yyyy-MM-dd HH:mm:ss.SSS}] [thread=%thread] [level=%-5level] [logger=%logger{15}] [requestID=%X{X-Request-Id}] - %msg %n"
```

Comment:
Can you change `src/test/resources/config/test-it-config.yaml` to match? Maybe without the timestamp, but the rest should probably be updated